src/pages/index.module.scss

```diff
@@ -5,3 +5,21 @@
 
 .main {
 }
+
+.text-glow {
```

Comment:
I've added glow of background color to text and a few more elements to make it look nicer if text intersects floating stars at some resolutions, e.g.
<details>

<summary>screenshot</summary>

![](https://i.imgur.com/lfeSYHZ.png)
</details>